notebook/API.livemd

```diff
@@ -11,8 +11,105 @@ base_url = IO.gets("Base URL") |> String.trim()
 ## Users
 
 ### GET /v1/me
+
 Used to fetch the current user's profile details
 
 ```elixir
 LivemanApi.get(base_url, "/v1/me")
 ```
+
+## Surveys
+
+### GET /v1/surveys
+
+Used to fetch all the available surveys.
+
+Returns a 200 OK response
+
+```elixir
+LivemanApi.get(base_url, "/v1/surveys")
+```
+
+Supports pagination by passing the page number and page size as params
+
+```elixir
+LivemanApi.get(base_url, "/v1/surveys", %{page: 2, page_size: 8})
+```
+
+### POST /v1/surveys
+
+Used to create an answer to a particular survey
+
+Requires a valid access_token in the header
+
+Returns a 201 Created response
+
+#### 201 OK
```

Comment:
Do you think we could have a sample json response, maybe with the markdown (code block) 🤔  I saw many API docs has it, not sure how it will look like here.

Reply:
It's possible to use a code block, but it would have the `Evaulate` button over it. I tried using a `json` markdown instead but for the response like `GET Surveys` its pretty big and it doesn't scroll like the code block for elixir does 😓